README.md

```diff
@@ -13,27 +13,6 @@ development inside a Bazel workspace.
 ## Table of Contents
 
 <!-- MARKDOWN TOC: BEGIN -->
-* [Documentation](#documentation)
```

Comment:
Seems like a bad change here

Reply:
Yeah. The issue is fixed in #552. Unfortunately, a recent change to stardoc is causing an issue with the legacy (non-bzlmod) tests in the root module. I am working #556 to isolate the non-bzlmod stuff to just the integration tests that need it. The rest of the repo will just be bzlmod enabled.